webpack/sharedPlugins.js

```diff
@@ -26,4 +27,10 @@ export const sharedPlugins = () => [
       return "cache-bust=" + Date.now();
     }`,
   }),
+
+  new PreloadWebpackPlugin({
+    rel: "preload",
+    as: "script",
+    include: "initial",
```

Comment:
Initial = just the scripts needed to render whatever initial landing page the user first requests. 

Looking at the output, its a lot! Really need to take a deep dive into our JS layer.